maven-plugin/src/main/java/org/openmrs/maven/plugins/packager/config/AbstractPackagerConfigMojo.java

```diff
@@ -133,6 +138,32 @@ public void savePropertiesToFile(Properties properties, File file) throws MojoEx
 		}
 	}
 
+	/**
+	 * Executes the maven dependency plugin, unpacking dependent artifacts into a standard directory structure
+	 * in the build directory so that these can be pulled in as a appropriate to the final configurations
+	 */
+	public void unpackDependency(ConfigDependency d, File unpackDir) throws MojoExecutionException {
+
+		getLog().info("Unpacking dependency to " + unpackDir);
+		executeMojo(
+				plugin("org.apache.maven.plugins", "maven-dependency-plugin", "3.1.1"),
+				goal("unpack"),
+				configuration(
+						element("artifactItems",
+								element("artifactItem",
+										element("groupId", d.getGroupId()),
+										element("artifactId", d.getArtifactId()),
+										element("version", d.getVersion()),
+										element("type", "zip"),
+										element("overWrite", "true"),
+										element("outputDirectory", unpackDir.getAbsolutePath())
+								)
+						)
+				),
+				getMavenExecutionEnvironment()
+		);
+	}
+
 	/**
```

Comment:
This is just moving the method here (which is removed below) into the superclass so that multiple Mojos can utilize it.